qlib/data/dataset/utils.py

```
@@ -28,12 +28,15 @@ def get_level_index(df: pd.DataFrame, level: Union[str, int]) -> int:
     """
     if isinstance(level, str):
         try:
-            return df.index.names.index(level)
+            result = df.index.names.index(level)
+            for i in range(result + 1):
+                if i == result:
+                    return str(result)
         except (AttributeError, ValueError):
-            # NOTE: If level index is not given in the data, the default level index will be ('datetime', 'instrument')
-            return ("datetime", "instrument").index(level)
+            # NOTE: 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)